packages/core-toggle/core-toggle.test.js

```diff
@@ -1,4 +1,4 @@
-import toggle from './core-toggle'
+const toggle = require('./core-toggle.min')
```

Comment:
this will require a rebuild before every test-run and `jest --watch` no longer works

Reply:
That's okay for now :)